src/auction.js

```diff
@@ -74,6 +74,11 @@ events.on(CONSTANTS.EVENTS.BID_ADJUSTMENT, function (bid) {
   adjustBids(bid);
 });
 
+const MAX_REQUESTS_PER_ORIGIN = 4;
```

Comment:
what's the magic behind 4? I thought modern browser was 6?

Reply:
On my specific test page there wasn't much difference with fast internet between 4 and 6 but there was a slight improvement on slow pages with 4 vs 6.  I'm not sure why, but I decided it would be safer to go with that value; if other's test prove differently then we could change it down the line.  It's also one of the reasons I made it configurable.